web/wwwroot/js/site.js

```diff
@@ -1,5 +1,25 @@
 ﻿let currentTab = 0;
-showTab(currentTab);
+
+// Get the input field
+let inputs, currentInput
+if (window.location.pathname === "/landlord/register" || window.location.pathname === "/landlord/edit") {
+    inputs = document.getElementsByTagName('input');
+    // Execute a function when the user presses a key on the keyboard
+    window.addEventListener("keyup", function (event) {
+        // If the user presses the "Enter" key on the keyboard
+        if (event.key === "Enter") {
+            // Cancel the default action, if needed
+            event.preventDefault();
+            // Deselects all boxes
+            for (currentInput = 0; currentInput < inputs.length; currentInput++) {
+                inputs[currentInput].blur();

```

Comment:
Is there a way to also deselect the next/previous buttons? I've noticed they stay highlighted after they're clicked.

Reply:
Yeah this should be pretty straightforward to put in!